pkg/registration/webhook/v1beta2/managedclusterset_conversion.go

```diff
@@ -15,9 +15,9 @@ ConvertTo is expected to modify its argument to contain the converted object.
 Most of the conversion is straightforward copying, except for converting our changed field.
 */
 // ConvertTo converts this ManagedClusterSet to the Hub(v1beta1) version.
-func (src *ManagedClusterSet) ConvertTo(dstRaw conversion.Hub) error {
+func (src *ManagedClusterSet) ConvertTo(logger klog.Logger, dstRaw conversion.Hub) error {
```

Comment:
we cannot change ConvertTo and ConvertFrom func's signature. It will break the webhook